tests/OpenRouterAPITest.php

```
@@ -1741,4 +1742,58 @@ public function it_sends_cache_control_in_request_body()
+        $payload = $chatData->convertToArray();
+
+        $this->mockOpenRouter($this->mockBasicBody());
+
+        /* EXECUTE */
+        $response = $this->api->chatRequest($chatData);
+
+        /* ASSERT */
+        $this->assertArrayHasKey('debug', $payload);
+        $this->assertEquals(['echo_upstream_body' => true], $payload['debug']);
+        $this->generalTestAssertions($response);
+    }
```

PHPMD (2.15.0)

[error] The method it_sends_debug_id_in_request_body is not named in camelCase. (undefined)

(CamelCaseMethodName)